ui/components/AutomationsTable.tsx

```diff
@@ -69,7 +69,7 @@ function AutomationsTable({ className, automations }: Props) {
           label: "Revision",
           value: "lastAttemptedRevision",
         },
-        { label: "Last Synced At", value: "lastHandledReconciledAt" },
+        { label: "Last Updated", value: "lastHandledReconciledAt" },
```

Comment:
I don't think this is accurate. Can you check the flux documentation to see what this field actually means?

Reply:
@sympatheticmoose 👀